Source/WebGPU/WGSL/GlobalVariableRewriter.cpp

```diff
@@ -1636,37 +1900,38 @@ static bool isExternalTexture(const AST::Variable& variable)
     return bindingTypeForType(variable.storeType()) == BindingType::TextureExternal;
 }
 
-static bool typesMatch(const AST::Variable& variable, const BindGroupLayoutEntry::BindingMember& bindingMember)
+static std::optional<String> errorValidatingTypes(const AST::Variable& variable, const BindGroupLayoutEntry::BindingMember& bindingMember)
```

Comment:
Does the empty String work for `std::nullopt` case?

Reply:
oh yes, we could use empty string instead